### PR TITLE
Date picker - IA update

### PR DESCRIPTION
**Pull request by chri2547**
image

### CODE REVIEW

**Comment by cyrichardson** [comment]
On it, @chri2547.

File: _docs/style/spacing.md [hunk 1]
```diff
@@ -266,7 +266,7 @@ Use a 24px line height in paragraphs that contain 16px font text. Add 16px spaci
 
 ## Component spacing
 
-Components such as buttons, text inputs, and date pickers are often placed beside each other on a control panel page. The Helix spacing system uses multiples of 4px as the baseline spacing value.
+Components such as buttons, text inputs, and date selector are often placed beside each other on a control panel page. The Helix spacing system uses multiples of 4px as the baseline spacing value.
```
Comment (cyrichardson) [inline _docs/style/spacing.md]: Should selector be plural like pickers was?

File: _component_design/date-selector.md [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+#### Calendar open and hover
+
+The date selector opens when the user clicks the date input field.
+
+Use the hover state when the user hovers over a date or a month navigation arrow. Hovering over the arrows indicates that they user can go back in time or ahead in time. Each date in the selector has a hover state, but only one date can be selected.
```
Comment (cyrichardson) [inline _component_design/date-selector.md]: that the user

File: _component_design/date-selector.md [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+#### Selection made
+
+The user can manually enter a date in the date text field, or select a date from the date selector. When the user selects a date, the date populates the date text field and the date selector closes.
```
Comment (cyrichardson) [inline _component_design/date-selector.md]: field or select
field, and the date

File: _component_design/time-picker.md [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+
+## Introduction
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### When to use
+
+Use the time picker if the list of times are sequential, such as 9 a.m., 10 a.m., 11 a.m., and so on.
```
Comment (cyrichardson) [inline _component_design/time-picker.md]: Use AM and PM for all a.m. and p.m. occurrences (per Style Guide)

File: _component_design/time-picker.md [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+
+Do not use the time picker if there are gaps between time options. For example, if 9 a.m., 12 p.m., and 3 p.m. are the options, use radio buttons.
+
+### Best practices
+
+- Times in a list should be incremental, usually by every 30 minutes or 1 hour, but 10- or 15-minute increments may be used depending on the use case.
```
Comment (cyrichardson) [inline _component_design/time-picker.md]: increments might

File: _component_design/time-picker.md [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Focused and expanded
+
+#### Predictive behavior
+
+- When the user enters a number on the keypad, the menu will jump to that number.
```
Comment (cyrichardson) [inline _component_design/time-picker.md]: the menu jumps to that number.

File: _component_design/time-picker.md [hunk 7]
```diff
@@ -0,0 +1,186 @@
...
+- For example, if the user types “7”, 7:00 p.m. will be scrolled to the top of the menu.
```
Comment (cyrichardson) [inline _component_design/time-picker.md]: 7:00 PM is scrolled

File: _component_design/time-picker.md [hunk 8]
```diff
@@ -0,0 +1,186 @@
...
+
+#### Default time shown
+
+- The default times shown in the drop-down menu start at 12:00 p.m.
```
Comment (cyrichardson) [inline _component_design/time-picker.md]: 12:00 PM.

File: _component_design/time-picker.md [hunk 9]
```diff
@@ -0,0 +1,186 @@
...
+- If specific times are available, the earliest one would show first, chronologically followed by later times.
+
+#### Default number of visible items
+
+- The number of items visible in the menu is four, unless the use case requires otherwise
```
Comment (cyrichardson) [inline _component_design/time-picker.md]: otherwise.

File: _component_design/time-picker.md [hunk 10]
```diff
@@ -0,0 +1,186 @@
...
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Idle and disabled
+
+- After the user has selected the time and shifted focus away, the time picker adopts its default state.
+- When disabled, the contents of the picker cannot be modified or triggered. Consider adding help text describing why the picker is disabled.
```
Comment (cyrichardson) [inline _component_design/time-picker.md]: When disabled, the user cannot modify or trigger the contents of the picker.

File: _component_design/date-selector.md [hunk 11]
```diff
@@ -0,0 +1,344 @@
...
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+### Behaviors
+
+Ensure that the date selector supports the following user interaction behaviors.
+
+**Note**: The start date receives focus when the user tabs onto it. Complete keyboard accessibility guidelines are forthcoming.
```
Comment (cyrichardson) [inline _component_design/date-selector.md]: are coming soon.

File: _docs/style/spacing.md (hunk 1 above)
Comment (chri2547) [reply _docs/style/spacing.md]: Yes, done.

File: _component_design/date-selector.md (hunk 2 above)
Comment (chri2547) [reply _component_design/date-selector.md]: Done.

File: _component_design/date-selector.md (hunk 3 above)
Comment (chri2547) [reply _component_design/date-selector.md]: Done.

File: _component_design/date-selector.md (hunk 11 above)
Comment (chri2547) [reply _component_design/date-selector.md]: Done.

File: _component_design/time-picker.md (hunk 10 above)
Comment (chri2547) [reply _component_design/time-picker.md]: Done.

File: _component_design/time-picker.md (hunk 9 above)
Comment (chri2547) [reply _component_design/time-picker.md]: Done.

File: _component_design/time-picker.md (hunk 8 above)
Comment (chri2547) [reply _component_design/time-picker.md]: Done.

File: _component_design/time-picker.md (hunk 7 above)
Comment (chri2547) [reply _component_design/time-picker.md]: Done.

File: _component_design/time-picker.md (hunk 6 above)
Comment (chri2547) [reply _component_design/time-picker.md]: Done.

File: _component_design/time-picker.md (hunk 5 above)
Comment (chri2547) [reply _component_design/time-picker.md]: Done.

File: _component_design/time-picker.md (hunk 4 above)
Comment (chri2547) [reply _component_design/time-picker.md]: Done.

**Pull request merged**